src/menu.rs

```diff
@@ -175,46 +283,46 @@ pub enum MenuItem {
   ///
   /// ## Platform-specific
   ///
-  /// - **Windows / Android / iOS:** Unsupported
+  /// - **Android / iOS:** Unsupported
   ///
   Separator,
 }
 
-impl MenuItem {
-  /// Create new custom menu item.
-  /// unique_menu_id is the unique ID for the menu item returned in the EventLoop `Event::MenuEvent(unique_menu_id)`
-  pub fn new(title: impl ToString) -> Self {
-    let title = title.to_string();
-    MenuItem::Custom(CustomMenu {
-      id: MenuId::new(&title),
-      name: title,
-      keyboard_accelerators: None,
-    })
+/// Custom menu item, when clicked an event is emitted in the EventLoop.
+/// You can modify the item after it's creation.
+#[derive(Debug, Clone)]
+pub struct CustomMenuItemHandle(pub CustomMenuItemPlatform);
```

Comment:
I think we should rename `CustomMenuItem` to `CustomMenuItemAttributes` or `MenuItemAttributes`, and then rename `CustomMenuItemHandle` to `CustomMenuItem` to follow the convention of this lib.

Reply:
I'm not sure about this;

```rust
submenu.add_item(MenuItemAttributes::new("Open new window", None, true, false));
```

or

```rust
submenu.add_item(CustomMenuItemAttributes::new("Open new window", None, true, false));
```

look a bit weird from my end

Reply:
What's about `MenuItemBuilder`

```rust
tray_menu.add_item(MenuItem::new("Focus window").with_selected(true));
```

rename `MenuItem` for `NativeItem` as it's used by `fn add_native_item`